docs/SECURITY.md

```diff
@@ -108,12 +127,92 @@ all:
 
 ## Transformations security
 
-Renditions using the LibreOffice transformer to render HTML are subject to the BSSRF attacks. In order to mitigate this risk, you may decide to disable the optimal renditions
-and instead generate lower quality renditions (where basically images would not be rendered as part of the HTML).
+Renditions using the LibreOffice transformer to render HTML are subject to the
+BSSRF attacks. In order to mitigate this risk, you may decide to disable the
+optimal renditions and instead generate lower quality renditions (where
+basically images would not be rendered as part of the HTML).
 That is achieved by the variable bellow:
 
 ```yaml
 bssrf_protection_enabled: true
 ```
 
 > This protection is disabled by default so users benefit from the best quality renditions.
+
+## Hosts certificates & keys
+
+By default the playbook will generate a dedicated host certificate and private
+key pair. It is possible to either use your own private PKI or let the playbook
+generate a small PKI infrastructure.
+If you want to leverage your own PKI infrastructure, you need to pass the
+playbook 2 assets:
+
+- A CA signing certificate in PEM format
+- The encrypted CA signing key in PEM format
```

Comment:
I am not sure this is really a common use case, most of the times the PKI is managed externally and who manage alfresco installs may not have access to CA pkey but just have a bunch of certificates generated externally (manually or not) after a pkey has been generated and a CSR submitted.

I understand that this approach is almost free with the implementation of a local PKI, but still may pretty uncommon.

Reply:
I think you're talking about a different use-case. When the pki is managed entirely externally (can still be internal to customer's org ofc) it is to possible for someone to generate its own certificates. And as a consequence the only thing one can get is the list of certificates. That's a tedious approach as for people managing the PKI but no generation is required at teh playbook level in this case (all certs are considered prereqs).

Reply:
ok in that case we could assume that all the certs will be placed in the same locations that the new playbook is handling 👍🏻

Reply:
will document that.